### DISCUSSION TITLE
Weird flickering when setting on medium brightness

### DISCUSSION THREAD

**Discussion by RHeynsZa** [arendst/Tasmota#22208] [Q&A]
This is really weird, so hoping someone else has run into this.

I have a [BNETA](https://templates.blakadder.com/bneta_IO-WIFI60-B22P.html) globe, pretty cheap old Tuya bulb. As soon as I have it on the white channel, and then swap to anywhere between 40 to 80% brightness, the bulb will start flickering

It doesnt ever restart, but just flickers randomly, every now and then. Not really sure what to do, I tried a few settings, no luck.

Also a weird thing I noticed, is it will never flicker on any RGB color, and also the white channel is significantly brighter than the RGB colors

**Comment by s-hadinger** [comment]
Flickering is usually a sign of the Power supply not handling the load. White Leds are typically much brighter than RGB which means consuming more.
